charts/chart-colors.js

```diff
@@ -0,0 +1,329 @@
+import { reverse } from '@dashboard/utils/array';
```

Comment:
I don't think we should be importing anything here. We should be able to do reverse on an array.